.github/workflows/release.yml

```diff
@@ -242,32 +239,26 @@ jobs:
 
       - name: Push testnet image
         run: |
-          docker tag cheqd-testnet ghcr.io/${{ github.repository_owner }}/${{ env.TESTNET_IMAGE_NAME }}:${{ env.VERSION }}
-          docker tag cheqd-testnet ghcr.io/${{ github.repository_owner }}/${{ env.TESTNET_IMAGE_NAME }}:latest
-          docker push ghcr.io/${{ github.repository_owner }}/${{ env.TESTNET_IMAGE_NAME }}:${{ env.VERSION }}
-          docker push ghcr.io/${{ github.repository_owner }}/${{ env.TESTNET_IMAGE_NAME }}:latest
+          docker tag cheqd-testnet ghcr.io/${{ github.repository_owner }}/cheqd-testnet:${{ env.VERSION }}
+          docker tag cheqd-testnet ghcr.io/${{ github.repository_owner }}/cheqd-testnet:latest
+          docker push ghcr.io/${{ github.repository_owner }}/cheqd-testnet:${{ env.VERSION }}
+          docker push ghcr.io/${{ github.repository_owner }}/cheqd-testnet:latest
```

Comment:
Are we going to change the name to localnet in this PR or a different PR?

Reply:
I suggest doing this in the next PR because most likely it will require actions from the GitHub admin panel. And this PR is big enough already.